packages/core/src/components/ModalNew/Modal/Modal.module.scss

```diff
@@ -1,17 +1,21 @@
 .overlay {
+  --monday-modal-z-index: 10000;
   position: fixed;
   inset: 0;
   background-color: rgba(41, 47, 76, 0.7);
+  z-index: var(--monday-modal-z-index);
 }
 
 .modal {
+  --monday-modal-z-index: 10000;
```

Comment:
You can use a scss variable for the value and assign it to a variable if that helps

Reply:
oh good point

Reply:
it actually on 2nd though won't work
as I cannot use the var in overlay if I set it in modal's div, and I cannot use it in modal's div if I set it in the overlay